### DISCUSSION TITLE
Not Zooming Into Bullet

### DISCUSSION THREAD

**Discussion by KarmaSenge** [vslinko/obsidian-outliner#319] [Q&A]
Hey all, I am having issues with Outliner zooming into a bullet. It won't do it at all. I am using Cybertron as my theme. Don't know if that has anything to do with it. I have tried with like 8 other themes. Not sure what to do. Thanks

**Comment by KarmaSenge** [comment]
?

**Comment by JacobOllivier** [comment]
Hi, I've just tried to use the zooming in functionality as well which I assumed worked similar to https://hogbaysoftware.netlify.app/bike/ but I'm also not able to figure out if if's not working as expected or if I'm missing something.

**Comment by Clastodon** [comment]
Hi there, I'm also struggling to make sense of it. I have tried changing Theme. I installed the LYT Kit 6, and it does exctly what you want it to do, why is my own graph not outlining. Maddeningly frustrating.

Here is the link to my thread on Discuss
https://forum.obsidian.md/t/outliner-plugin-doesnt-work/39223

And here is a screeshot of my Outline
![Untitled](https://user-images.githubusercontent.com/107256314/175121233-5c100167-6c65-472d-bd77-b9c540905a4c.png)

**Comment by Clastodon** [comment]
I made some progress over on the Discuss forum. It turned out I was using the tab key, but you have to put a 'dash' in there. If you follow the link it might give you an idea
